index.html

```diff
@@ -116,4 +112,12 @@ <h2>Testimonials</h2>
             <p><strong class="author">Lauren</strong></p>
         </span>
     </div>
+</div>
+
+<!-- Photos -->
+<div class="section-2">
```

Comment:
might be good to also use the 
```
<div class="details-container">
    <div class="details-item">
```
here as well, or mimic how the testimonials handle multiple items side-by side